suites/gcp.xml

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="GCP">
+    <test name="SmokeTest">
+        <classes>
+            <class name="io.managed.services.test.SmokeTest"/>
+        </classes>
+    </test>
+    <test name="SSOAuthTest">
+        <classes>
+            <class name="io.managed.services.test.SSOAuthTest"/>
+        </classes>
+    </test>
+    <test name="TestBase">
+        <classes>
+            <class name="io.managed.services.test.TestBase"/>
+        </classes>
+    </test>    
+    <test name="DeploymentsTest">
```

Comment:
reserved deployment will not be running against GCP without mod. this class first

Reply:
Removed Deployments Test